pkg/config/app/kuma-cp/kuma-cp.defaults.yaml

```diff
@@ -29,10 +29,19 @@ store:
     driverName: pgx # ENV: KUMA_STORE_POSTGRES_DRIVER_NAME
     # Connection Timeout to the DB in seconds
     connectionTimeout: 5 # ENV: KUMA_STORE_POSTGRES_CONNECTION_TIMEOUT
-    # Maximum number of open connections to the database
+    # MaxConnectionLifetime (applied only when driverName=pgx) is the duration since creation after which a connection will be automatically closed
+    maxConnectionLifetime: "1h" # ENV: KUMA_STORE_POSTGRES_MAX_CONNECTION_LIFETIME
+    # MaxConnectionLifetimeJitter (applied only when driverName=pgx) is the duration after maxConnectionLifetime to randomly decide to close a connection.
+    # This helps prevent all connections from being closed at the exact same time, starving the pool.
+    maxConnectionLifetimeJitter: "0s" # ENV: KUMA_STORE_POSTGRES_MAX_CONNECTION_LIFETIME_JITTER
+    # HealthCheckInterval (applied only when driverName=pgx) is the duration between checks of the health of idle connections.
+    healthCheckInterval: "1m" # ENV: KUMA_STORE_POSTGRES_HEALTH_CHECK_INTERVAL
+    # MinOpenConnections (applied only when driverName=pgx) is the minimum number of open connections to the database
+    minOpenConnections: 0 # ENV: KUMA_STORE_POSTGRES_MIN_OPEN_CONNECTIONS
+    # MaxOpenConnections is the maximum number of open connections to the database
     # `0` value means number of open connections is unlimited
     maxOpenConnections: 50 # ENV: KUMA_STORE_POSTGRES_MAX_OPEN_CONNECTIONS
-    # Maximum number of connections in the idle connection pool
+    # MaxIdleConnections (applied only when driverName=postgres) is the maximum number of connections in the idle connection pool
     # <0 value means no idle connections and 0 means default max idle connections
```

Comment:
Should 0 means no idle and not have a default?

Reply:
I think we do overrides well (we take a config initialized with Default and then override with whatever comes from yaml/envvars) but could double check.

Reply:
I don't think I should be changing the meaning of this because it's already implemented and could be a breaking change for someone. Maybe when we completely remove `pq` as a driver?